frontend/src/Commons/Forms/InputModules/FieldsAndSoil/FieldsAndSoil.style.ts

```diff
@@ -73,5 +74,41 @@ const StyledButtonGroupContainer = styled.div`
     width: 67px;
   }
 `;
+const StyledTestContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+const StyledRadioGroupContainer = styled.div`
+  display: flex;
+
+  @media (min-width: ${screenSizes.desktop}) {
+    width: 50%;
+  }
```

Comment:
I think you might be a little better off just using a fixed size for this container. Not really a need for resizing your Yes/No gap